src/main/java/com/google/sps/data/PlaceReviews.java

```diff
@@ -15,9 +16,9 @@
  */
 public class PlaceReviews {
 
-  private final String place_id;
-  private ArrayList<Comment> reviews;
-  private HashSet<String> reviewers;
+  public final String place_id;
+  public List<Comment> reviews;
+  private List<String> reviewers;
```

Comment:
Going to delete reviewers field anyway.

Reply:
I had done that so that I could access it by index in parallel with the reviews so I could actually associate the two, but that's obsolete since we're getting rid of it in the next iteration.